src/homework/homework02/Person.java

```diff
@@ -1,41 +1,41 @@
 package homework.homework02;
 
 public class Person {
-    public int getPesel() {
-        return pesel;
-    }
 
-    public void setPesel(int pesel) {
+    protected int pesel;
+    protected String name;
+    protected String surname;
```

Comment:
Po co one są protected?

Reply:
Z rozpędu? Powinny być public?

Reply:
-1 do enkapsulacji (hermetyzacji)
A po co? Będzie zachęcać do grzebania w nich z pominięciem przewidzianych getterów/setterów

Reply:
To jak nie protected i nie public, to wybór jest mały...